src/gui/update.rs

```diff
@@ -544,6 +575,9 @@ pub fn handle_message(ajour: &mut Ajour, message: Message) -> Result<Command<Mes
         Message::RuntimeEvent(iced_native::Event::Window(
             iced_native::window::Event::Resized { width, height },
         )) => {
+            let width = (width as f64 * ajour.scale_state.scale) as u32;
+            let height = (height as f64 * ajour.scale_state.scale) as u32;
+
```

Comment:
This tripped me up so much. Apparently once you apply scaling, the Resized event returns a "scaled" width and height. So I had to adjust it back before saving. I was getting tripped up why the window would open at different sizes anytime I changed the scale.